Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM debian:11.4 as base
+FROM debian:11.5 as base
 
-LABEL maintainers="pastakhov@yandex.ru,alexey@wikiteq.com"
+LABEL maintainers="pavel@wikiteq.com,alexey@wikiteq.com"
 LABEL org.opencontainers.image.source=https://github.com/WikiTeq/Taqasta
 
-ENV MW_VERSION=REL1_35 \
-	MW_CORE_VERSION=1.35.8 \
+ARG COMPOSER_TOKEN
```

Comment:
this apparently reverts the change made on Taqasta https://github.com/WikiTeq/Taqasta/pull/34 for Composer tokens during CI builds. The `COMPOSER_TOKEN` was dropped in favor of mounted secrets at https://github.com/WikiTeq/Taqasta/pull/34 so if merged we will have a degradation on this subject

Need to be changed to match https://github.com/WikiTeq/Taqasta/pull/34

Reply:
Also see line https://github.com/WikiTeq/Taqasta/pull/26/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R799 , it looks like we're having a mix of both here

Reply:
good catch, I have no idea how I missed this.